solutions/0000-0099/0026 - Remove Duplicates from Sorted Array-easy.md

```diff
@@ -0,0 +1,92 @@
+---
+description: 'Author: @pankajsirari222| https://leetcode.com/problems/remove-duplicates-from-sorted-array/'
+tags: [Array, Two Pointers]
+---
+
+# 0026 - Remove Duplicates from Sorted Array (Easy) 
+
+## Problem Link
+
+https://leetcode.com/problems/remove-duplicates-from-sorted-array/
+
+## Problem Statement
+
+Given an integer array nums sorted in non-decreasing order, `remove the duplicates` in-place such that each unique element appears only once.
```

Comment:
Problem statement format is not the same as mentioned in LC. 

Use correct format, bold it, use LaTeX and backtick wherever required. Otherwise Reader's find difficulty by reading this. Few keywords gives a hint to approach the problem.